generator/typescript/dropbox_types.d.tstemplate

```diff
@@ -29,7 +29,7 @@ declare module DropboxTypes {
      * @param state State that will be returned in the redirect URL to help
      *   prevent cross site scripting attacks.
      */
-    getAuthenticationUrl(redirectUri: string, state?: string, authType = 'token'): string;
+    getAuthenticationUrl(redirectUri: string, state?: string, authType?: 'token' | 'code'): string;
```

Comment:
This fixes https://github.com/dropbox/dropbox-sdk-js/issues/199

Reply:
Oh I see! thank you very much!